### ISSUE TITLE
Error: getaddrinfo ENOTFOUND api.noopschallenge.com

### ISSUE THREAD

**Issue by goscha01** [noops-challenge/hexbot#24]
sent get request to https://api.noopschallenge.com/hexbot?count=5 and got an error.

**Comment by iamserda** [comment]
Yeah, I just found out about this project and tested the endpoint on insomnia, it timed out. root domain isn't available either.

**Comment by chewett** [comment]
@doolittle @davidmfoley Do you know why this is down?

**Comment by doolittle** [comment]
It appears that there's been a DNS change beyond our control. What the DNS formerly pointed to is still running; we've reached out to GitHub about a fix.

**Comment by chewett** [comment]
Many thanks!

**Comment by eallenOP** [comment]
Just want to add my support for this project and hope it gets fixed. Extremely useful for teaching!

**Comment by yoerriwalstra** [comment]
any news on this issue? Still receiving an error when sending a request to the API :(
```
➜  pathbot curl -X POST https://api.noopschallenge.com/pathbot/start
curl: (6) Could not resolve host: api.noopschallenge.com
```

**Comment by cturland** [comment]
Is this a dead project now?
